ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py

```diff
@@ -68,6 +68,41 @@ def test_jax_numpy_absolute(
     )
 
 
+# argmax
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        force_int_axis=True,
+        min_num_dims=1,
+        min_axis=-1,
```

Comment:
Is there any specific reason we need to limit the axis? If not, use `valid_axis=True` instead.